modules/navier_stokes/doc/content/source/postprocessors/MfrWeightedAverage.md

```diff
@@ -0,0 +1,34 @@
+# VolumetricFlowRate
+
+!syntax description /Postprocessors/MfrWeightedAverage
```

Comment:
Mfr stands for Mass Flow Rate and yet this is Mass Flux weighted.

so let s MassFluxWeightedAverage, spelled out

Reply:
actually it s also a flow rate

so it's MassFluxWeightedFlowRate ?